ipn/local.go

```diff
@@ -788,6 +797,21 @@ func routerConfig(cfg *wgcfg.Config, prefs *Prefs, dnsDomains []string) *router.
 	return rs
 }
 
+func wgCIDRsToFilter(cidrs ...[]wgcfg.CIDR) (ret []filter.Net) {
```

Comment:
document this? (the `...` along with the `[]` caught me off guard at first. but it's fine.)

Reply:
Maybe call it `cidrSets`, though? It's double plural, really.